error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-01 16:46:57  [ main:1 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-01 16:48:21  [ main:1 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-01 16:48:34  [ main:0 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-01 16:48:47  [ main:1 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-01 16:50:36  [ main:1 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-01 16:54:38  [ main:1 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-01 16:57:50  [ main:1 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
2021-07-02 12:00:24  [ main:1 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
